pkg/controller/resourcecleanup/resourcecleanup_controller.go

```diff
@@ -218,9 +219,18 @@ func (r *ReconcileResourceCleanup) Cleanup(ctx context.Context, cluster *cluster
 	if len(works.Items) == 1 && works.Items[0].Name == klusterletCRDWorkName {
 		// the manifestWorks are deleted by registration controller.
 		// the agent may be orphaned if the CRD manifestWork is force deleted directly.
-		if works.Items[0].DeletionTimestamp.IsZero() {
+		crdWork := works.Items[0]
+		if crdWork.DeletionTimestamp.IsZero() {
+			return r.clientHolder.WorkClient.WorkV1().ManifestWorks(cluster.Name).
```

Comment:
I recall that the registration controller is responsible for deleting this ManifestWork. Why is the import-controller also deleting it?

Reply:
you are right. do not need delete again here. 
fixed.